src/lib/components/editor/util.ts

```diff
@@ -20,7 +20,7 @@ export const initEditor = (monacoEditor): void => {
 			'journey'
 		],
 		keywords: ['participant', 'as'],
-		arrows: ['---', '===', '-->', '==>', '->>', '->'],
+		arrows: ['---', '===', '-->', '==>', '->>', '-->>', '->', '-)', '--)', '-x', '--x'],
```

Comment:
Did you validate if these were rendered in browser?
I had tried adding few extra arrows and they didn't work.

Reply:
It might have been happening because the + or - is present. I am going to update the PR to include that.